material_3_demo/web/index.html

```diff
@@ -38,22 +38,30 @@
   </script>
   <!-- This script adds the flutter initialization JS code -->
   <script src="flutter.js" defer></script>
+  <script type="text/javascript" src="assets/packages/web_startup_analyzer/lib/web_startup_analyzer.js"></script>
 </head>
 <body>
-  <script>
-    window.addEventListener('load', function(ev) {
-      // Download main.dart.js
-      _flutter.loader.loadEntrypoint({
-        serviceWorker: {
-          serviceWorkerVersion: serviceWorkerVersion,
-        },
-        onEntrypointLoaded: function(engineInitializer) {
-          engineInitializer.initializeEngine().then(function(appRunner) {
-            appRunner.runApp();
-          });
-        }
-      });
+<script>
+  var flutterWebStartupAnalyzer = new FlutterWebStartupAnalyzer();
```

Comment:
This rename feels...weird. Could we just have one variable?

Reply:
Good catch, I don't know why I wrote it like this.

Reply:
Actually this is needed as a global variable so that the Dart code can talk to it.